cmd/ctr/commands/pprof/pprof.go

```diff
@@ -56,10 +56,17 @@ var Command = cli.Command{
 var pprofGoroutinesCommand = cli.Command{
 	Name:  "goroutines",
 	Usage: "dump goroutine stack dump",
+	Flags: []cli.Flag{
+		cli.StringFlag{
+			Name:  "debug",
+			Usage: "debug pprof args",
+			Value: "2",
+		},
+	},
 	Action: func(context *cli.Context) error {
 		client := getPProfClient(context)
 
-		output, err := httpGetRequest(client, "/debug/pprof/goroutine?debug=2")
+		output, err := httpGetRequest(client, fmt.Sprintf("/debug/pprof/goroutine?debug=%s", context.String("debug")))
```

Comment:
How about using IntFlag?

While it may be harmless, the flag can be used to inject any arbitrary query parameters currently.